Day0/krischal111/Assignment1/solution.c

```diff
@@ -0,0 +1,257 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <math.h>
+
+#define SQUARE(anything) (anything*anything)
+#define renderfile "render.txt"
+#define resultfile "test.ppm"
+
+#define circlescale 5
+#define ppmscale 25
+#define ppmshow 1
+#define circleshow 0
+#define rectangleshow 0
+#define circledetails 0
+#define rectangledetails 0
+
+/* Structures used  */
+
+struct coordinate
+{
+    int x, y;
+};
+
+struct pixel
+{
+    unsigned char r, g, b;
+};
+
+struct rectangle
+{
+    struct coordinate start, size;
+    struct pixel colour;
+};
+
+struct circle
+{
+    struct coordinate centre;
+    int radius;
+    struct pixel colour;
+};
+
+struct box
+{
+    struct coordinate start, end;
+};
+
+/* Functions used*/
+int squareof(int number)
+{
+    return number*number;
+}
+
+char  writepixel(void * image_data, struct coordinate size, struct coordinate coord, struct pixel pxl)
```

Comment:
Why do you return the same same from the function always?

Reply:
Made the function void type.